pkg/visor/visorconfig/v1.go

```diff
@@ -100,15 +100,15 @@ type V1UptimeTracker struct {
 	Addr string `json:"addr"`
 }
 
-// V1AppDisc configures Skywire App Discovery Clients.
-type V1AppDisc struct {
+// V1Disc configures Skywire App Discovery Clients.
+type V1Disc struct {
```

Comment:
`ServiceDisc` maybe better? We have three discoveries at this point overall.

Reply:
Okay